lib/ansible/modules/system/seboolean.py

```diff
@@ -294,39 +314,42 @@ def main():
     if not get_runtime_status(ignore_selinux_state):
         module.fail_json(msg="SELinux is disabled on this host.")
 
-    name = module.params['name']
+    names = module.params['name']
     persistent = module.params['persistent']
     state = module.params['state']
 
     result = dict(
-        name=name,
+        name=names[0] if len(names) == 1 else names,
         persistent=persistent,
         state=state
     )
     changed = False
 
-    if hasattr(selinux, 'selinux_boolean_sub'):
-        # selinux_boolean_sub allows sites to rename a boolean and alias the old name
-        # Feature only available in selinux library since 2012.
-        name = selinux.selinux_boolean_sub(name)
+    for pos, name in enumerate(names):
+        if hasattr(selinux, 'selinux_boolean_sub'):
+            # selinux_boolean_sub allows sites to rename a boolean and alias the old name
+            # Feature only available in selinux library since 2012.
+            names[pos] = selinux.selinux_boolean_sub(name)
 
-    if not has_boolean_value(module, name):
-        module.fail_json(msg="SELinux boolean %s does not exist." % name)
+    for name in names:
+        if not has_boolean_value(module, name):
+            module.fail_json(msg="SELinux boolean %s does not exist." % name)
 
-    if persistent:
-        changed = semanage_boolean_value(module, name, state)
-    else:
-        cur_value = get_boolean_value(module, name)
-        if cur_value != state:
-            changed = True
-            if not module.check_mode:
-                changed = set_boolean_value(module, name, state)
-                if not changed:
-                    module.fail_json(msg="Failed to set boolean %s to %s" % (name, state))
-                try:
-                    selinux.security_commit_booleans()
-                except Exception:
-                    module.fail_json(msg="Failed to commit pending boolean %s value" % name)
+    for name in names:
+        if persistent:
+            changed = semanage_boolean_value(module, name, state)
```

Comment:
I'd want to see the role stay atomic and have either everything or nothing change.  If the module fails, nothing should change on the system.  That might either mean moving the loop into the `semanage_boolean_value` function, or supporting externally-triggered commit for the function.